test/unit/test_main.py

```diff
@@ -101,3 +103,4 @@ def test_main_with_json_str(
         mock_connection, McpServerSettings.model_validate(settings_json)
     )
     mock_server.run.assert_called_once()
+    mock_install_model.assert_called_once_with(settings_json["language_model"])
```

Comment:
The assert_called functions are usually not optimal for pytest tests. Better is usually `call(..) inbmock_install_model.mock_calls`

Reply:
Can you please elaborate on this and/or point to the source of information? I am not aware of any issues with `assert_called_once_with`.

Reply:
OK, then it's fine. Here, it is really more like a formality check. To make the sonar happy :)